data-loader/core/src/main/java/com/scalar/db/dataloader/core/dataimport/dao/ScalarDbDao.java

```diff
@@ -59,9 +50,7 @@ public Optional<Result> get(
 
     try {
       Get get = createGetWith(namespace, table, partitionKey, clusteringKey);
-      Optional<Result> result = storage.get(get);
-      logger.info(String.format(GET_COMPLETED_MSG, loggingKey));
```

Comment:
I think there is another option to change the log level to DEBUG or TRACE. What do you think?